types/image-import/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "private": true,
+    "name": "@types/image-import",
+    "version": "1.0.9999",
+    "nonNpm": true,
```

Comment:
There's no such package `image-import`. This package seems to be designed as some sort of shorthand for declaring these file types, but that kind of package isn't something we accept on DT. https://github.com/DefinitelyTyped/DefinitelyTyped#what-about-type-definitions-with-no-matching-package

Reply:
(This is something that a tool like vite automatically does for you.)